src/pages/components/menu.mdx

```diff
@@ -141,22 +141,53 @@ import MenuSizeCode from '!!raw-loader!../../examples/dropdowns/menu/Size.tsx';
 
 ## API
 
+A Menu expects the following overall structure:
+
+```jsx isLight
+<Dropdown>
+  <Trigger>
+  <Menu>
+    <Item>
+    <Item>
+    ...
+  </Menu>
+</Dropdown>
+```
+
+<!-- markdownlint-disable no-emphasis-as-heading -->
+
+**Implementation notes**
+
+<!-- markdownlint-enable no-emphasis-as-heading -->
+
+If loading is asynchronous, consider delaying Item rendering to prevent Downshift from reapplying accessibility and
+interaction attributes.
```

Comment:
I'm having a difficult time painting a picture of this scenario. How would a consumer delay `Item` rendering?

Reply:
Plus 1: this phrasing re: reapplication of a11y attributes seen throughout feels off and is perhaps connected to an earlier component version that didn't handle conditional rendering as well as the current one does.

Reply:
Okay, this seems unnecessary in that case. Removed!